packages/business-rules-lib/src/validators/__tests__/date.validators.spec.js

```diff
@@ -0,0 +1,95 @@
+import Joi from 'joi'
+import * as dateValidation from '../date.validators.js'
+
+describe('date validators', () => {
+  describe('dayValidator', () => {
+    it.each([1, 15, 31])('expects an integer between 1 and 31', async value => {
+      await expect(dateValidation.createDayValidator(Joi).validateAsync(value)).resolves.toEqual(value)
+    })
+
+    it.each([
+      [0, 'must be greater than or equal to 1'],
+      [32, 'must be less than or equal to 31'],
+      [1.5, 'must be an integer'],
+      ['foo', 'must be a number'],
+      ['', 'must be a number'],
+      [null, 'must be a number'],
+      [undefined, 'is required']
+    ])('throws an error if the value is invalid', async (value, message) => {
```

Comment:
With a test description like this, we'll only have an output of 'throws an error if the value is invalid' seven times, which isn't very informative. A description of 'When date value is %s validator throws an error of %s' would probably do the trick.

This goes for all the other it.each tests in this file, if you don't mind? Thanks

Reply:
Sorted!